### DISCUSSION TITLE
Tutorial vs. Real World access?

### DISCUSSION THREAD

**Discussion by JaeEmAre67** [pnp/pnpcore#903] [Q&A]
We're writing an Azure Function App to read data from SharePoint sites/subsites. I know the tutorial shows "FullControl" permissions are needed. Is that just for the tutorial (since it's creating) or does our Function App also need FullControl just to read content?

**Comment by jansenbe** [comment]
@JaeEmAre67 : that's just the tutorial, if you're only reading data, you do have enough with read permissions.
